controller/backing_image_manager_controller.go

```diff
@@ -789,6 +789,8 @@ func (c *BackingImageManagerController) createBackingImageManagerPod(bim *longho
 	if err != nil {
 		return err
 	}
+	types.AddGoCoverDirToPod(podManifest)
```

Comment:
Suggest moving this to `generateBackingImageManagerPodManifest` consistent with other changes here.

Reply:
Good catch. Updated it. Thank you.